lib/tokenlist.cpp

```diff
@@ -292,6 +294,9 @@ Token *TokenList::copyTokens(Token *dest, const Token *first, const Token *last,
 
             links.pop();
         }
+
+        column += tok->next() ? tok->next()->column() - tok->column() : tok->str().length() + 1;
```

Comment:
I am not so sure about this. At least when we replace tokens it's better that the column just match the replaced token. imagine MY_INT generates a warning the warning should point at MY_INT not at some other location later..